shared/js/project.js

```diff
@@ -464,6 +496,13 @@ function executeCallback(callback, data) {
 }
 
 /**
+ * is the current project (if any) editable by the logged in user (if any)?
+ */
+function isEditable() {
```

Comment:
It doesn't look like this will enforce `frozen` on the `/projects` overview page, so users could still delete it from there.  Maybe it would be better to have `frozen` be a separate `:state` column value? (like 'active' or 'deleted')